src/Sentry.Unity/Integrations/SceneManagerIntegration.cs

```diff
@@ -29,7 +29,8 @@ void OnSceneManagerOnSceneLoaded(SceneAdapter scene, LoadSceneMode mode)
 
             hub.AddBreadcrumb(
                 message: $"Scene '{scene.Name}' was loaded",
-                category: "scene.loaded");
+                category: "scene.loaded",
+                type: "navigation");
```

Comment:
navigation breadcrumbs should also have `data`. From the develop docs:

navigation
A navigation event can be a URL change in a web application, or a UI transition in a mobile or desktop application, etc.
Internally we display breadcrumbs of type default that contain category navigation as a breadcrumb of type navigation.

Its data property has the following sub-properties:

from (Required)

A string representing the original application state / location.

to (Required)

A string representing the new application state / location.

```JSON
{
  "type": "navigation",
  "category": "navigation",
  "timestamp": "2016-04-20T20:55:53.845Z",
  "data": {
    "from": "/login",
    "to": "/dashboard"
  }
}
```

Reply:
Just thinking out loud here: You can load scenes additionally, how'd that work with the `from`->`to`?

Reply:
could be `current` vs `previous` ? it can be anything that makes sense here

Reply:
then, I guess it's not a navigation event

Reply:
That'd actually make sense. As in: Which scene cause which other scene to load.

Reply:
Sounds good - doesn't work. The `fromScene` in the `ActiveSceneChanged` call is always `null`. In all Unity versions.